### DISCUSSION TITLE
accessing files in dir. gives error.

### DISCUSSION THREAD

**Discussion by magic-thomas** [sveltejs/kit#6340] [Q&A]
I just had read new Doc. 

ignore this.

-----------------------------------------------------------------------------

I installed sveltekit app.

```sh
$ npm list |grep svelte

├── @sveltejs/adapter-auto@1.0.0-next.66
├── @sveltejs/kit@1.0.0-next.442
├── svelte-check@2.8.1
├── svelte-preprocess@4.10.7
├── svelte@3.49.0

```

I have files 

```sh
src/routes/videos/+page.svelte
src/routes/videos/+page.ts
src/routes/videos/show.svelte
```

Accessing <code> http://localhost:5173/videos </code> is fine .

But when I access <code>http://localhost:5173/videos/show</code>

it says <code>404: Not found: /videos/show</code>

How can I access show.svelte ?

**Comment by mustofa-id** [comment]
That should be `src/routes/videos/show/+page.svelte`. Think new routing system like directory based routing.
